.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: build and consume by foreman & plugins
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - repo: foreman
+            org: theforeman
+          - repo: katello
+            org: Katello
+          - repo: foreman-tasks
+            org: theforeman
+    steps:
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
+      # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm install` in plugins.
+      - uses: ruby/setup-ruby@v1
```

Comment:
Instead of updating the postinstall script, I think something like that should work `npm install --ignore-scripts`

Reply:
Looks like it wasn't needed after all :)